index.test-d.ts

```diff
@@ -1,12 +1,14 @@
 import {expectType, expectError} from 'tsd';
 import tempy = require('.');
 
-const options: tempy.Options = {};
-expectType<string>(tempy.directory());
-expectType<string>(tempy.file());
-expectType<string>(tempy.file({extension: 'png'}));
-expectType<string>(tempy.file({name: 'afile.txt'}));
-expectError(tempy.file({extension: 'png', name: 'afile.txt'}));
-expectType<string>(tempy.root);
-expectType<string>(tempy.writeSync('hi there'));
-(async () => {expectType<string>(await tempy.write('hi there'))});
+(async () => {
+    const options: tempy.Options = {};
+    expectType<string>(tempy.directory());
+    expectType<string>(tempy.file());
+    expectType<string>(tempy.file({extension: 'png'}));
+    expectType<string>(tempy.file({name: 'afile.txt'}));
+    expectError(tempy.file({extension: 'png', name: 'afile.txt'}));
+    expectType<string>(tempy.root);
+    expectType<string>(tempy.writeSync('hi there'));
+    expectType<string>(await tempy.write('hi there'));
+})
```

Comment:
This is not top-level await. `await` is not at the top-level. You can drop the `(async () => {`, which is also invalid as it's missing `})` => `})()`.